holoviews/operation/normalization.py

```diff
@@ -175,3 +177,38 @@ def _normalize_raster(self, raster, key):
                 if range:
                     norm_raster.data[:,:,depth] /= range
         return norm_raster
+
+
+class subcoordinate_group_ranges(Operation):
+    """
+    Compute the data range group-wise in a subcoordinate_y overlay,
+    and set the dimension range of each Chart element based on the
+    value computed for its group.
+
+    This operation is useful to visually apply a group-wise min-max
+    normalisation.
+    """
+
+    def _process(self, overlay, key=None):
+        vmins = defaultdict(list)
+        vmaxs = defaultdict(list)
+        for el in overlay:
+            if not isinstance(el, Chart):
+                continue
+            vmin, vmax = el.range(1)
+            vmins[el.group].append(vmin)
+            vmaxs[el.group].append(vmax)
+
+        minmax = {
+            group: (np.min(vmins[group]), np.max(vmaxs[group]))
+            for group in vmins
+        }
+        new = []
+        for el in overlay:
+            if not isinstance(el, Chart):
+                new.append(el)
+                continue
+            y_dimension = el.vdims[0]
+            y_dimension = y_dimension.clone(range=minmax[el.group])
+            new.append(el.redim(**{y_dimension.name: y_dimension}))
+        return overlay.clone(data=new)
```

Comment:
Might suggest that you check if any of the layers were actually Charts and then make this operation a complete no-op if none of them were, i.e. simply return `overlay` as is without cloning it.

Reply:
Implemented in https://github.com/holoviz/holoviews/pull/6124/commits/b3965b4dbcdaaa0410812b8a9918c9436f90935b